EASY-1320 custom vagrant without stripped versions of bags from act-easy

jo-pol opened the pull request:
Fixes EASY-1230

#### When applied it will

* have a local VM in the project
* [x] make the solr schema (duplicated from dtap) avaialble for resue in dtap
* [x] not use too real bags for test data

#### Where should the reviewer @DANS-KNAW/easy start?

#### How should this be manually tested?

In the project root
* `mvn clean install`
* `vagrant up`
* `curl http://test.dans.knaw.nl:8983/solr/fileitems/query\?q\=%2A%3A%2A`
  it should find nothing
* `vagrant ssh`
* `easy-update-solr4files-index init`
  it should report it indexed one bag for the pdbs store and nothing for the other stores
* `exit`
* The curl command above should now find 9 files,
  to reduce output extend the query with `\&fl=id\&rows\=1`
  or execute the default query with the browser on
  `http://test.dans.knaw.nl:8983/solr/#/fileitems/query`
* another `vagrant provision` should skip copying test bags as it take a while

The dev-tool scripts still run with deasy.


#### Related pull requests on github
repo                       | PR                | note
-------------------------- | ----------------- | ----
dans-                    | ...  | ...

PaulBoon reviewed (commented):
Please change the 'real' test data to something synthetic. 
Otherwise we might get into trouble with the 'owners' of the data.

jo-pol committed:
custom vagrant, copied solr schema from dtap

https://github.com/DANS-KNAW/easy-dtap/blob/c9d345d7fbb26b1daa0126cb570a68043eb7e27c/provisioning/roles/easy_solr_search2/files/fileitems/schema.xml

jo-pol committed:
skip copy bag if done before

jo-pol committed:
TODO's
